### PR TITLE
[MNOE-890] Reseller's products on landing page

### PR DESCRIPTION
**Pull request by enizor**
* Add products to the /marketplace to get them while not authenticated
* Add local products to the settings/landing page

### CODE REVIEW

**Comment by enizor** [comment]
@ouranos

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/marketplace_controller.rb [hunk 1]
```diff
@@ -35,6 +35,16 @@ def index
         apps
       end
 
+    query = MnoEnterprise::Product.includes(:'values.field', :assets, :categories, :product_pricings, :product_contracts).where(active: true)
```
Comment (ouranos) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/marketplace_controller.rb]: Indentation

File: api/lib/mno_enterprise/concerns/controllers/jpi/v1/marketplace_controller.rb [hunk 2]
```diff
@@ -35,6 +35,16 @@ def index
...
+
+    # Ensure prices include organization-specific markups/discounts
+    if params[:organization_id] && parent_organization_id
```
Comment (ouranos) [inline api/lib/mno_enterprise/concerns/controllers/jpi/v1/marketplace_controller.rb]: No need to check `params[:organization_id]` it's already checked in `parent_organization_id`

File: core/app/models/mno_enterprise/tenant_config.rb [hunk 3]
```diff
@@ -157,12 +157,18 @@ def self.update_locales_list!
     # TODO: replace with a Proc.call
     def self.update_application_list!
```
Comment (ouranos) [inline core/app/models/mno_enterprise/tenant_config.rb]: This gonna need a big refactor at some point ;)

**Comment by MAhsenArif** [comment]
This can be closed as task is continued on https://github.com/maestrano/mno-enterprise/pull/656

**Comment by ouranos** [comment]
To be continued on #656

**Pull request closed without merging**